docs/rst/spelling_wordlist.txt

```diff
@@ -40,6 +40,7 @@ NetX
 Nucleo
 NuttX
 Pico
+preallocates
```

Comment:
Can we take the opportunity to sort this file correctly. There are two instances of turtlesim 😖